### PR TITLE
Fixing port name list option.

### PR DESCRIPTION
**Pull request by evgenyfedoruk**
"name" should be specified instead of "display_name".
Link to the issue: https://github.com/ContainX/openstack4j/issues/963

### CODE REVIEW

**Comment by auhlig** [comment]
Hi @evgenyfedoruk,
Thanks investigating and submitting this PR.
As the attribute was `display_name` before and changed to `name` later, would it make sense to have both methods in our lib? So how about keeping the displayName method and just adding the following?
```
public PortListOptions name(String name) {
        return add("name", name);
    }
```

**Comment by evgenyfedoruk** [comment]
Hi @auhlig
You right, pushed an update with new method and old method deprecated.

File: core/src/main/java/org/openstack4j/model/network/options/PortListOptions.java [hunk 1]
```diff
@@ -61,13 +61,26 @@ public PortListOptions adminState(boolean adminState) {
...
+     * neutron port does not have attribute named "display_name".
+     * Use {@link #name(String)} to filter ports by name
+     *
+     * @param displayName the port display name
      * @return options
      */
+    @Deprecated
     public PortListOptions displayName(String displayName) {
-        return add("display_name", displayName);
+        return add("name", displayName);
```
Comment (auhlig) [inline core/src/main/java/org/openstack4j/model/network/options/PortListOptions.java]: And this one stays `"display_name"`, not?

Comment (evgenyfedoruk) [inline core/src/main/java/org/openstack4j/model/network/options/PortListOptions.java]: Oh.. sure.

**Comment by auhlig** [comment]
Many thanks for contributing @evgenyfedoruk.

**Pull request merged**